modules/sdk/src/main/scala/org/tessellation/sdk/config/types.scala

```diff
@@ -69,4 +70,9 @@ object types {
     defaultCheckAttempts: PosInt,
     ensureCheckInterval: FiniteDuration
   )
+
+  case class CollateralConfig(
+    userAmount: Option[Amount],
```

Comment:
Who is `user`? I mean we do not treat nodes as users so we should unify the glossary.
Node operators are not using the node but are hosting/operating the node.

Reply:
Changed to `mainnetAmount`.